ytdl_nfo/Ytdl_nfo.py

```diff
@@ -8,7 +8,7 @@ def __init__(self, file_path, extractor=None):
         self.dir = os.path.dirname(file_path)
 
         # Read json data
-        with open(self.path, 'r') as f:
+        with open(self.path, "rt", encoding="utf-8") as f:
```

Comment:
This fixes an issue where unicode characters in video info can result in an exception